include/zephyr/bluetooth/bluetooth.h

```diff
@@ -2250,6 +2250,15 @@ static inline int bt_le_whitelist_clear(void)
 /**
  * @brief Set (LE) channel map.
  *
+ * Channel map format is chan_map[5] = {0xXX, 0xXX, 0xXX, 0xXX, 0xXX};
+ * chan_map[0] = 0-7
+ * chan_map[1] = 8-15
+ * chan_map[2] = 16-23
+ * chan_map[3] = 24-31
+ * chan_map[4] = 32-39
+ *
+ * A bit set in this array disables the specified channel.
```

Comment:
Isn't it the opposite? 

![image](https://github.com/zephyrproject-rtos/zephyr/assets/2520413/770ea914-cb52-4502-a964-4df401645c0c)

Setting a bit to 0 marks is "bad", 1 as "unknown"